FT019 - feat: added routing paths

damiankubecki opened the pull request:
In auth.module.ts and home.module.ts I added objects with paths related to corresponding modules

damiankubecki committed:
feat: added routing paths

In auth.module.ts and home.module.ts I added objects with paths related to corresponding modules

damiankubecki commented:
Po konsultacji z zespołem zdecydowaliśmy, aby obiekt ze ścieżkami znajdował się wewnątrz pliku auth.module.ts/home.module.ts zamiast w osobnym pliku.

akademiaEnvelo commented on src/app/features/home/home.module.ts:
```diff
@@ -2,11 +2,37 @@ import { NgModule } from '@angular/core';
+  notifications: 'notifications',
+  requestBox: 'request-box',
+  meender: 'meender',
+};
```
oznaczmy object literal jako `as const`, dzięki temu będziemy mieli bardziej wyizolowane typy

akademiaEnvelo commented on src/app/features/auth/auth.module.ts:
```diff
@@ -2,11 +2,15 @@ import { NgModule } from '@angular/core';
 
+export const paths = {
+  default: '',
+};
```
jak niżej :)

Commit:
feat: updated routing paths

This commit renames objects with home and auth paths and adds a new home paths with examples

akademiaEnvelo commented on src/app/features/home/home.module.ts:
```diff
@@ -2,14 +2,65 @@ import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
 import { HomeComponent } from './home.component';
 
+export const homePaths = {
```
```typescript
export const homePaths = {
  //...
} as const;
```
sprawdź jakie są różnice w typie homePaths przed i po dodaniu `as const`

akademiaEnvelo replied:
dobra praktyką jest też zapisywanie słowników lub stałych w zapisie `UPPERCASED`

```
const HOME_PATHS = {
 DEFAULT: ''
}
```

dzięki temu widzimy w kodzie że zmienna której używamy jest stałą wartością nie zmieniająca się w czasie.

Commit:
feat: updated routing paths

This commit makes paths objects declared as const and makes their keys uppercased
